requirements.txt

```diff
@@ -0,0 +1,4 @@
+-e git+https://github.com/open-telemetry/opentelemetry-python.git@937863f8294d6347e131eaaedbc1b877598b13d9#egg=skiptargetinstall-opentelemetry-api&subdirectory=opentelemetry-api
```

Comment:
One thing that sucks about this solution is that pip must install __the whole repo___ when it just wants to install a target in a sub directory...

The best solution I’ve found is that we could instead [install from a zip file](https://stackoverflow.com/a/24811490). That would require us to update the release format on the Contrib package to release the sub targets (api, SDK, proto, instrumentation, test/utils) as zip files.